AirCasting/CreateSessionViews/SelectPeripheralView.swift

```diff
@@ -62,15 +60,19 @@ struct SelectPeripheralView: View {
                         connectButton.disabled(true)
                     }
                 }
+                .onAppear {
+                    if CBCentralManager.authorization == .allowedAlways {
+                        _ = bluetoothManager.centralManager
```

Comment:
oh dear :) let's leave it, but please put a comment with explanation why it's needed and what it does